connectors/src/admin/cli.ts

```diff
@@ -264,7 +264,7 @@ const slack = async (command: string, args: parseArgs.ParsedArgs) => {
       if (!connector) {
         throw new Error(`Could not find connector for workspace ${args.wId}`);
       }
-      await throwOnError(enableSlackBot(connector.id));
+      await throwOnError(toggleSlackbot(connector.id, true));
```

Comment:
What is this diff ?

Reply:
The enableSlackbot method's signature has been changed to toggleBot accepting a boolean so that it can be used in cli.ts as well as in the connectors api by bot_enabled.ts (via the switches you suggested in connectors/index.ts)